R/pkg/R/mllib_regression.R

```diff
@@ -110,7 +125,8 @@ setClass("IsotonicRegressionModel", representation(jobj = "jobj"))
 #' @seealso \link{glm}, \link{read.ml}
 setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
           function(data, formula, family = gaussian, tol = 1e-6, maxIter = 25, weightCol = NULL,
-                   regParam = 0.0, var.power = 0.0, link.power = 1.0 - var.power) {
+                   regParam = 0.0, var.power = 0.0, link.power = 1.0 - var.power,
+                   stringIndexerOrderType = "frequencyDesc") {
```

Comment:
actually, I mean it as https://github.com/actuaryzhang/spark/blob/66bc786add41df52baead5a7d38b0b6b035d764d/R/pkg/R/mllib_clustering.R#L167

but then we will need to tweak it to have a default value